### DISCUSSION TITLE
DES phase activation

### DISCUSSION THREAD

**Discussion by astro-kevin** [flybywiresim/aircraft#8151] [Help] [answered]
On the beginners guide, it seems to imply that the DES phase will activate automatically when in cruise phase and a lower altitude is selected. But…I just tried that and the phase does not change, so what’s going on here?

**Comment by ExampleWasTaken** [comment, marked as answer]
In order for the DES phase to activate you have to actually descent. To initiate the descent select your target altitude and either press or pull the knob.
